### DISCUSSION TITLE
laravel reverb and pulse. Pulse does not show connections

### DISCUSSION THREAD

**Discussion by basbroens** [laravel/framework#51117] [Q&A]
Hi All,

I recently installed Laravel Reverb and Pulse.

I added: 

   <livewire:reverb.connections cols="6" />
    <livewire:reverb.messages cols="6" />

To the pulse view and in the config for pulse:
 'recorders' => [
    ReverbConnections::class => [
            'sample_rate' => 1,
        ],

        ReverbMessages::class => [
            'sample_rate' => 1,
        ],
]

Reverb is working correctly and i see a realtime graph for messages. However connections stays empty.

Does anyone know why?

**Comment by basbroens** [comment]
never mind php artisan pulse:check was the answer

**Comment by Saifallak** [comment]
What package/card do you use to show reverb statuses?
